src/20240723_AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnershipPhase2/TokenLogicKarpatkeyServiceProviderPartnershipPhase2.md

```diff
@@ -0,0 +1,130 @@
+# TokenLogic + karpatkey Service Provider - Phase II
+
+---
+
+Title: TokenLogic + karpatkey - Phase II
+Author: TokenLogic karpatkey
+Date: 2024-07-29
+
```

Comment:
```suggestion
title: TokenLogic + karpatkey - Phase II
author: TokenLogic karpatkey
discussions: https://governance.aave.com/t/arfc-tokenlogic-karpatkey-financial-service-providers-phase-ii/18285
snapshot: https://snapshot.org/#/aave.eth/proposal/0xc44ec840f8f7f6ca3ef2f2a4289882c4cdc1a8b3e6e9ad6b811a640097a8016a
```

@LucasWongC Should be formatted correctly as otherwise would break the UI: https://vote.onaave.com/proposal-create-overview/?ipfsHash=QmbQmeAtXdcuUeUkhTGoTxjcWRDgXN8BoqocryxUdeQ98Y